pkg/ipam/types.go

```diff
@@ -145,6 +145,6 @@ func (p Pool) String() string {
 	return string(p)
 }
 
-const (
-	PoolDefault Pool = ipamOption.PoolDefault
+var (
+	PoolDefault = Pool(option.Config.IPAMDefaultIPPool)
```

Comment:
I don't think this will work. This will be initialized at package load time, i.e. before flags are parsed and `option.Config.*` is populated. So `PoolDefault` will always contain the default value of `option.Config.IPAMDefaultIPPool`.

Reply:
Updated. @tklauser